### PR TITLE
fix(configs): don't follow symlinks for loaders

### PR DESCRIPTION
**Pull request by PanayotCankov**

### CODE REVIEW

File: prepublish/common/exports.js [hunk 1]
```diff
@@ -49,6 +49,11 @@ module.exports = env => {
             // and will enable us to work with symlinked packages during development.
             symlinks: false
         },
+        resolvePlugin: {
```
Comment (sis0k0) [inline prepublish/common/exports.js]: Isn't that property called `resolveLoader`?

Comment (PanayotCankov) [reply prepublish/common/exports.js]: :+1:

Comment (PanayotCankov) [reply prepublish/common/exports.js]: Forcepushed,
I won't merge these before we meet at the office ;)

**Comment by sis0k0** [comment]
js

**Pull request merged**